src/common/services/api/radioStations.service.js

```diff
@@ -5,7 +5,8 @@ import 'rxjs/add/operator/mergeMap'
 import 'rxjs/add/operator/pluck'
 import 'rxjs/add/observable/of'
 import 'rxjs/add/observable/throw'
-import appConfig from 'common/app.config'
```

Comment:
This removal should be squashed with 2e34bf2ebe49cf9c7e5545066e8edf07ca785808 or perhaps 2e34bf2ebe49cf9c7e5545066e8edf07ca785808 should be removed altogether.